src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  categories: [],
+  status: false,
+};
+
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState,
+  getCategoryStatus: () => 'This page is under construction',
+});
```

Comment:
> - [ ] Please update the categories slice for the check status action it should return the string "Under Construction"  and In case of unknown action, it should return the current state as requested in the project requirements.👍
> - You can consider the below code.
> ```
>  reducers: {
>     getCategoryStatus: (state, action) => {
>       state.categories =
>         action.payload === "Under construction"
>           ? "Under construction"
>           : state.categories
>     },
>   },
> ```
> 

Kindly remember to export the getCategoryStatus action:

`export const { getCategoryStatus} = categoriesSlice.actions`